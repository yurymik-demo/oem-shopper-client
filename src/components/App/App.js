import React, { Component } from "react";
import { BrowserRouter, Route } from "react-router-dom";
import Header from "../Header/Header";
import Footer from "../Footer/Footer";
import HomeRoute from "../../routes/home/home";

class App extends Component {
  render() {
    return (
      <BrowserRouter>
        <div className="grey lighten-5">
          <Header />
          <div className="container ">
            <Route exact path="/">
              <HomeRoute />
            </Route>
          </div>
          <Footer />
        </div>
      </BrowserRouter>
    );
  }
}

export default App;
