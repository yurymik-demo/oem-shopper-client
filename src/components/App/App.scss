.app-container {
  min-height: 600px;
}
