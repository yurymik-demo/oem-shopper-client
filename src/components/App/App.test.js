import React from "react";
import ReactDOM from "react-dom";
import App from "./App";

jest.mock("../../routes/home/home", () => {
  return () => "home_route_component";
});

jest.mock("../Header/Header", () => {
  return () => "<Header />";
});

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  expect(div).toMatchSnapshot();
  ReactDOM.unmountComponentAtNode(div);
});
