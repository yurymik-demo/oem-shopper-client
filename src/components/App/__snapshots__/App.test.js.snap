// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    class="grey lighten-5"
  >
    &lt;Header /&gt;
    <div
      class="container app-container"
    >
      home_route_component
    </div>
    <footer
      class="page-footer teal"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col l6 s12"
          >
            <h5
              class="white-text"
            >
              OEM Store
            </h5>
            <p
              class="grey-text text-lighten-4"
            >
              To learn more about who we are click on these links:
            </p>
          </div>
          <div
            class="col l4 offset-l2 s12"
          >
            <h5
              class="white-text"
            >
              More About Us:
            </h5>
            <ul>
              <li>
                <a
                  class="grey-text text-lighten-3"
                  href="/#!"
                >
                  About
                </a>
              </li>
              <li>
                <a
                  class="grey-text text-lighten-3"
                  href="/#!"
                >
                  Careers
                </a>
              </li>
              <li>
                <a
                  class="grey-text text-lighten-3"
                  href="/#!"
                >
                  Investors
                </a>
              </li>
              <li>
                <a
                  class="grey-text text-lighten-3"
                  href="/#!"
                >
                  Partners
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        class="footer-copyright"
      >
        <div
          class="container"
        >
          Â© 2018 Copyright Text
          <a
            class="grey-text text-lighten-4 right"
            href="/#!"
          >
            More Links
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
